spec/unit/plugins/linux/filesystem2_spec.rb

```diff
@@ -52,6 +52,12 @@
     end
   end
 
+  it "sets both filesystem and filesystem_v2 attributes" do
+    plugin.run
+    expect(plugin[:filesystem]).to_not be_nil
+    expect(plugin[:filesystem_v2]).to_not be_nil
```

Comment:
`expect(plugin[:filesystem]).to eq(plugin[:filesystem_v2])` as well maybe?

but this test also seems to be failing and filesystem_v2 seems to be nil for some reason...